terraform/pubsub/bind/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "name" { value = google_service_account.account.name }
```

Comment:
If the aim is to match the legacy broker as closely as possible, would `Name` and `Email` be better?